docs/getting-started-guides/kubeadm.md

```diff
@@ -195,6 +195,48 @@ Once a pod network has been installed, you can confirm that it is working by che
 
 And once the `kube-dns` pod is up and running, you can continue by joining your nodes.
 
+If you see the following status 
+```
```

Comment:
Formatting: add a newline above this line to add some space between the sentence and the console output.